dkc/core/tests/test_folder_rest.py

```diff
@@ -51,7 +95,7 @@ def test_folder_rest_retrieve(admin_api_client, folder):
 
 @pytest.mark.django_db
 def test_folder_rest_update(admin_api_client, folder):
-    resp = admin_api_client.put(
+    resp = admin_api_client.patch(
```

Comment:
We might want to outright disallow `PUT` requests at some point, but since I think we typically expect clients to use `PATCH` requests, it's better for this test to use that.